css/styles.css

```diff
@@ -0,0 +1,81 @@
+body {
```

Comment:
This file contain too much of custom styles and they are styles that can achieved with pure bootstrap classes